includes/frontend.php

```diff
@@ -535,9 +535,7 @@ public function register_styles() {
 
 		$direction_suffix = is_rtl() ? '-rtl' : '';
 
-		$frontend_base_file_name = $this->is_optimized_css_mode() ? 'frontend-lite' : 'frontend';
-
-		$frontend_file_name = $frontend_base_file_name . $direction_suffix . $min_suffix . '.css';
+		$frontend_file_name = "frontend{$direction_suffix}{$min_suffix}.css";
```

Comment:
Ok, so are we removing `frontend-lite`? 🤔

Reply:
YES!